### ISSUE TITLE
Interaction with PneumaticCraft: Repressurized causes crash with RFTools Powercells

### ISSUE THREAD

**Issue by Flanks255** [McJty/RFTools#1719]
MC: 1.12.2
Forge: 14.23.3.2673
RFTools: 7.33
MCJtyLib: 2.6.7

When using a Pneumatic Helmet, with the block tracker, with the rf block tracker enabled, looking at a RFTools powercell, with a link card in it causes the following NPE in RFTools:

[https://pastebin.com/irxH3rsY](https://pastebin.com/irxH3rsY)

Related Pnc ticket where desht has some ideas: [https://github.com/TeamPneumatic/pnc-repressurized/issues/157](https://github.com/TeamPneumatic/pnc-repressurized/issues/157)

**Comment by McJty** [comment]
PneumatiCraft is trying to access RF from client side. The RFTools powercells don't allow this. I can fix the crash but reading RF client-side is not going to work in any case so doing this is most likely also an issue on Pneumaticraft side

**Comment by desht** [comment]
Yeah, I don't need you to add clientside reading, but it would be nice if it at least didn't crash if you tried to read client side :)

I have some ideas on fixing this on the PneumaticCraft side anyway (custom energy level request/response packets, similar to how you handle Waila for the Powercell block).
